/*
 * This file was automatically generated by EvoSuite
 * Mon May 27 07:44:54 CST 2019
 */

package app.server.util;

import static org.evosuite.runtime.EvoAssertions.verifyException;
import static org.junit.Assert.fail;

import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.Rule;
import org.junit.Test;
import org.junit.rules.ExpectedException;
import org.junit.runner.RunWith;

import app.server.bean.Comment;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;

@RunWith(SpringRunner.class)
@SpringBootTest
public class PtoVTest{
    @Rule
    public ExpectedException thrown = ExpectedException.none();

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      thrown.expect(NullPointerException.class);
      Comment comment0 = new Comment();
      PtoV ptoV0 = PtoV.ptoV;
      // Undeclared exception!
      ptoV0.getCommentVO(comment0);

  }
}
